python/pyproject.toml

```diff
@@ -30,7 +30,7 @@ pyspark = "^3.4"
 numpy = ">= 1.7"
 
 [tool.poetry.group.dev.dependencies]
-black = "^25.1.0"
+black = "^23.12.1"
```

Comment:
why do you downgrade?

Reply:
https://github.com/apache/spark/blob/master/dev/pyproject.toml#L30

Reply:
ah.. thanks.